Duplicati/UnitTest/CommandLineOperationsTests.cs

```diff
@@ -82,23 +82,43 @@ private void DownloadS3FileIfNewer(string destinationFilePath, string url)
             try
             {
                 // check if the file should be downloaded, exception if not
-                using ((HttpWebResponse)webRequest.GetResponse())
+                using (var wr=(HttpWebResponse)webRequest.GetResponse())
 
                 using (WebClient client = new WebClient())
-                {
-                    client.DownloadFile(url, destinationFilePath);
+                { // try to workaround a weird intermittent bug where downloaded size

```

Comment:
These files are quite large, and likely error prone. 

Is the reason we download them all the time so that automated checks can work?

I've considered making their existence and sha256 a requirement for the test to run successfully.

Overall I don't like the surprise that running tests can download nearly 1GB of data without you knowing that's going to happen.